src/commands/new.ts

```diff
@@ -501,7 +505,7 @@ export default {
     // #endregion
 
     // #region Cache dependencies
-    if (shouldFreshInstallDeps && cacheExists === false) {
+    if (shouldFreshInstallDeps && cacheExists === false && useCache) {
```

Comment:
Even though `useCache` is `false` by default now, this was still getting copied. Added `useCache` check to only copy if `--use-cache` is `true`